draft-ietf-cose-bls-key-representations.md

```diff
@@ -200,6 +200,32 @@ E'(GF(p^8))
 
 {backmatter}
 
+# Appendix
+
+## Example JSON Web Key Representations
+
+The following example JWK is for the Bls12381 curve where the public key is in the G1 subgroup.
+
+```
+{
+  "kty": "OKP",
+  "crv": "Bls12381G1",
+  "x": "olbufORO-VTYtXkD2k7hXc45KJONpCudrOAeIpOH3Wqu2tJ9MNUNcTze3Eqkr5dp",
+  "d": "_nmYLctt4EA3jtzM_DoO72zOgJtk_7pDqXoFCfpizFE"
```

Comment:
While it is possible to share a common private key and generate two public keys across the two sub-groups this violates common practise around key reusage, I will update the example to have distinct key pairs. @selfissued, do you think we should offer a security consideration about this case, I'm not aware of any particular insecurity that this would create, however it would algorithm dependent.

Reply:
I think it's worth mentioning the tradeoffs, at least at the level you're describing them in this comment.  It may cause people who know more than we do to weigh in.